packages/relay/src/lib/eth.ts

```diff
@@ -164,13 +184,13 @@ export class EthImpl implements Eth {
         return EthImpl.feeHistoryZeroBlockCountResponse;
       }
 
-      let feeHistory: object | undefined = cache.get(constants.CACHE_KEY.FEE_HISTORY);
+      let feeHistory: object | undefined = this.cache.get(constants.CACHE_KEY.FEE_HISTORY);
       if (!feeHistory) {
 
         feeHistory = await this.getFeeHistory(blockCount, newestBlockNumber, latestBlockNumber, rewardPercentiles, requestId);
 
         this.logger.trace(`${requestIdPrefix} caching ${constants.CACHE_KEY.FEE_HISTORY} for ${constants.CACHE_TTL.ONE_HOUR} ms`);
-        cache.set(constants.CACHE_KEY.FEE_HISTORY, feeHistory, constants.CACHE_TTL.ONE_HOUR);
+        this.cache.set(constants.CACHE_KEY.FEE_HISTORY, feeHistory);
```

Comment:
With the new logic does one have to create a new cache object if the max time period is not the default 1 hour set in options?
Or is it flexible enough to pass in a time param to override the default option?

Reply:
Yes, according to their documentation it's possible to pass options, which override default ones: https://github.com/isaacs/node-lru-cache#setkey-value--size-sizecalculation-ttl-nodisposeonset-start-